lib/collection/url.js

```diff
@@ -347,7 +355,8 @@ _.extend(Url, /** @lends Url */ {
         if (_.isString(p.host)) {
             url = url.substr(p.host.length); // remove host from url
 
-            if (p.auth = _.get(p.host.match(regexes.extractHostAuth), GET_1)) {
+            // eslint-disable-next-line no-cond-assign
+            if (p.auth = _.get(p.host.match(regexes.extractHostAuth), GET_1)) { // ignore the single `=`
```

Comment:
Following the pattern on [line 355](https://github.com/postmanlabs/postman-collection/pull/111/files/54aee3331279e079dd344c925d8b683e3f846147#diff-283827426a2d44cafd2ef3dd38b5157fR355), I'll be separating the assignment to outside the if block condition.